selfdrive/controls/lib/alerts_offroad.json

```diff
@@ -40,5 +40,13 @@
   "Offroad_StorageMissing": {
     "text": "Storage drive not mounted.",
     "severity": 1
+  },
+  "Offroad_CarUnrecognized": {
+    "text": "openpilot was unable to identify your car. Your vehicle has new firmware versions that aren't recognized. Add new firmware to the proper vehicle. Need help? Join discord.comma.ai.",
```

Comment:
we should expand on these. perhaps we link to a wiki page or README?

Reply:
```suggestion
    "text": "openpilot was unable to identify your car. Your vehicle's ECUs are not recognized. Please submit a pull-request to add the firmware versions to the proper vehicle. Need help? Join discord.comma.ai.",
```